### DISCUSSION TITLE
404 Error after Custom Domain set up

### DISCUSSION THREAD

**Discussion by ghost** [community/community#32598] [Other Feature Feedback, Questions, & Ideas]
I set up a custom domain for my website and now it is giving a 404 error. It was working fine before I set up the custom domain. 
![Screen Shot 2022-09-10 at 2 25 05 PM](https://user-images.githubusercontent.com/88013012/189502297-99bb57a4-8b0a-4892-953a-10d1a040c282.png)

I have tried many things:
* Updating the CNAME file to contain `https://stefanimoore.com` instead of just `stefanimoore.com`
* Updated `homepage` property in the `package.json` file to `https://stefanimoore.com`, even tried just `http://stefanimoore.com` and same results. 
* Have checked and my repo is using `gh-pages` branch with the folder `/docs` selected. 

Here is my DSN setup in case it is something wrong with that:
![Screen Shot 2022-09-10 at 2 13 32 PM](https://user-images.githubusercontent.com/88013012/189502114-441699d4-f8c2-44d9-853f-11a00e45e789.png)

The command I use to publish is: `gh-pages -d build` and that seems to be working fine as it I see the new commits in the `gh-pages` branch. 

All the solutions that came up on my searchers for this particular error did not work for me. 

Here is the link to my [repo](https://github.com/steafelipe/stmoor-website ) in case it is helpful.

**Comment by samuel-ma** [comment]
@steafelipe, which domain registrar did you buy your domain from?
The domain process is pretty simple actually 
Make sure you've configured the DNS transfer first your domain host

**Reply by ghost** [reply]
I used Google Domain. What do you mean configure DNS transfer?

**Comment by samuel-ma** [comment]
Ok, and UAH stands for?
